docs/configuration.md

```diff
@@ -5,16 +5,19 @@ For non-core options (for example UI options), take a look at [Settings](https:/
 
 ### Base options
 
-|Option|Description|Default|
-|---|---|---|
-|`APP_NAME`|The gallery name|`Lychee`|
-|`APP_ENV`|Environment of your gallery, `production` or `development`|`production`|
-|`APP_URL`|The URL of your gallery (which resolves to the `public/` folder)|`http://localhost`|
-|`APP_KEY`|Your app key which is used for encryption (set during installation)|`null`|
-|`TIMEZONE`|The timezone of your photos, requires a named timezone identifier like `Europe/Paris`, don't use an offset (`+01:00`) or an abbreviation (`CEST`)|system timezone of server|
-|`LYCHEE_UPLOADS`|Path to uploads directory|`uploads/` inside `public/`|
-|`LYCHEE_UPLOADS_URL`|URL to uploads directory|`uploads/`|
-|`TRUSTED_PROXIES`|Trusted proxy IP addresses|`null`|
+| Option     | Description                                                                                                     | Default            |
+|------------|-----------------------------------------------------------------------------------------------------------------|--------------------|
+| `APP_NAME` | The gallery name                                                                                                | `Lychee`           |
+| `APP_ENV`  | Environment of your gallery, `production` or `development`                                                      | `production`       |
+| `APP_URL`  | The URL of your gallery (which resolves to the `public/` folder)                                                | `http://localhost` |
+| `APP_DIR`  | The URL path after the domain name of your gallery (which resolves to the `public/` folder), better left empty. | _empty_            |
+| `APP_KEY`  | Your app key which is used for encryption (set during installation)                                             | `null`             |
+|`TIMEZONE`  | The timezone of your photos, requires a named timezone identifier like `Europe/Paris`, don't use an offset (`+01:00`) or an abbreviation (`CEST`)|system timezone of server|
+|`LYCHEE_UPLOADS`    | Path to uploads directory                                                                  | `uploads/` inside `public/` |
+|`LYCHEE_UPLOADS_URL`| URL to uploads directory, better left empty                                                | `/uploads`                  |
+|`TRUSTED_PROXIES`   | Trusted proxy IP addresses                                                                 | `null`                      |
+|`LIVEWIRE_ENABLED`  | Enable v5 Livewire frontend, set to false if you prefer old v4 (but no front-end updates). | `true`                      |
```

Comment:
This makes it sound like it's something we are genuinely planning to support as an option forever. I would set the expectations more realistically:

```suggestion
|`LIVEWIRE_ENABLED`  | Enable v5 Livewire frontend; set to false if the new frontend doesn't work for you (this is a temporary workaround). | `true`                      |
```